app/forms/declaration/DeclarationAdditionalActors.scala

```diff
@@ -27,6 +27,8 @@ case class DeclarationAdditionalActors(eori: Option[Eori], partyType: Option[Str
 
   import DeclarationAdditionalActors._
 
+  def id: String = s"${partyType.getOrElse("")}-${eori.getOrElse("")}"
```

Comment:
Is this id needed now that you are using the hashcode-based id?
Or the `buildObjectInstanceFromId` method below?

Reply:
I have removed it from the code. Thank you for spotting it